src/System.Net.Primitives/tests/FunctionalTests/CredentialCacheTest.cs

```diff
@@ -94,6 +98,37 @@ private static IEnumerable<CredentialCacheCount> GetCredentialCacheCounts()
             yield return CreateHostPortCredentialCacheCount();
             yield return CreateUriAndHostPortCredentialCacheCount();
         }
+        
+        public static IEnumerable<object[]> StandardAuthTypeWithNetworkCredential =>
+            new[]
+            {
+                new object[] {authenticationTypeNTLM, credential1},
+                new object[] {authenticationTypeKerberos, credential2},
+                new object[] {authenticationTypeNegotiate, credential3},
+                new object[] {authenticationTypeBasic, credential4},
+                
+                new object[] {authenticationTypeNTLM, CredentialCache.DefaultNetworkCredentials as NetworkCredential},
+                new object[] {authenticationTypeKerberos, CredentialCache.DefaultNetworkCredentials as NetworkCredential},
+                new object[] {authenticationTypeNegotiate, CredentialCache.DefaultNetworkCredentials as NetworkCredential},
+                // Default credentials cannot be supplied for the Basic authentication scheme.
```

Comment:
>While in .Net Core, WDigestAvailable will always be false (we don't support it). Therefore, it will always throw ArgumentException. So this is a behavior difference, if put them in the same test case, my concern is it may fail in some .Net Framework conditions.

In that case, you write your test like this:

```c#
if (PlatformDetection.IsFullFramework)
{
   // .NET Framework behavior
}
else
{
   // .NET Core behavior
}
```